frontend/src/citizen-frontend/applications/editor/validations.ts

```diff
@@ -105,17 +105,31 @@ export const validateApplication = (
           terms
         )
       ),
+      connectedDaycarePreferredStartDate:
+        apiData.type === 'PRESCHOOL' &&
+        featureFlags.preschoolApplication.serviceNeedOption
```

Comment:
The new `connectedDaycarePreferredStartDate` feature flag should be used here. Apart from this everything looks good now!

Reply:
Fixed.